app/src/androidTest/assets/test.html

```diff
@@ -26,6 +26,7 @@ <h2>Service worker</h2>
 <p id="status"></p>
 
 <script type="text/javascript">
+        localStorage.setItem('focusLocalStorageTest', 'testValue');
```

Comment:
nit: can you add a comment explaining why we do this?

Reply:
Also, it'd be great to add a newline below this to separate the intentions between the two sections of code.